spec/services/submit_to_app_store/prior_authority/event_builder_spec.rb

```diff
@@ -376,6 +402,8 @@
           expect(listed_corrections).to eq ['alternative_quote_2']
         end
       end
+
+      context 'when further info'
```

Comment:
??? can this be removed or are you expecting additional tests

Reply:
i think I wrote a whole test and didn't save the file :S